File: demo/app/app/src/main/java/com/vectordemo/domain/model/oss/OssBatchUploadModel.kt
package com.vectordemo.domain.model.oss

import com.vectordemo.utils.json.GsonBean

data class OssBatchUploadModel(
    val userId: Long,
    val bucketName: String,
    val successCount: Int,
    val failCount: Int,
    val items: List<OssUploadItemModel>
) : GsonBean

File: demo/flutter/flutternew/lib/domain/model/oss/oss_batch_upload_model.dart
import 'oss_upload_item_model.dart';

class OssBatchUploadModel {
  final int userId;
  final String bucketName;
  final int successCount;
  final int failCount;
  final List<OssUploadItemModel> items;

  const OssBatchUploadModel({
    required this.userId,
    required this.bucketName,
    required this.successCount,
    required this.failCount,
    required this.items,
  });
}
